openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -64,6 +65,7 @@ class MayaPluginInfo(object):
     # Include all lights flag
     RenderSetupIncludeLights = attr.ib(
         default="1", validator=_validate_deadline_bool_value)
+    StrictErrorChecking = attr.ib(default="1")
```

Comment:
Should it be string or int?
```suggestion
    StrictErrorChecking = attr.ib(default=1)
```